### PR TITLE
Adds audit block and code block

### PR DESCRIPTION
**Pull request by nykka**
Pivotal ticket reference: https://www.pivotaltracker.com/story/show/170808137

This PR:

- Adds the Audit warning/advisories block styles.
- Adds styles for `pre` and `code`.

Screen Shot 2020-01-28 at 14 01 36
Screen Shot 2020-01-28 at 14 01 44

### CODE REVIEW

**Review by ABizzinotto — APPROVED** [review]
Looks good to me. @etagwerker could you check before I merge? Thanks!

File: vendor/assets/stylesheets/ombulabs/styleguide/4-molecules/_code.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  padding: 0 10px;
+  text-align: left;
+}
+code{
+  @include code;
+}
+pre{
+  code{
+    @include code;
+  }
```
Comment (etagwerker) [inline vendor/assets/stylesheets/ombulabs/styleguide/4-molecules/_code.scss]: @nykka This looks like a bit of over-engineering, why not just use a simple CSS rule?

For example: 

```
code, pre code {
  background-color: $lighter-grey;
  // ... 
}
```

**Review by etagwerker — COMMENTED** [review]
@nykka Looks good, just one comment about implementation of SCSS.

File: vendor/assets/stylesheets/ombulabs/styleguide/4-molecules/_code.scss (hunk 1 above)
Comment (nykka) [reply vendor/assets/stylesheets/ombulabs/styleguide/4-molecules/_code.scss]: Makes sense!

**Review by etagwerker — APPROVED** [review]

**Pull request merged**